src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -50,9 +52,20 @@ export default class CourtOrderPoa extends Vue {
   private planOfArrangement = false
   private valid = false
 
-  @Watch('validate')
+  /** Clear rules and reset validations. */
+  private clearValidations (): void {
+    this.courtOrderNumRules = []
+    this.$refs.courtNumRef.resetValidation()
```

Comment:
You didn't need to update the local IF declaration for this to work?

BTW, I like this validation design.

Reply:
Hmm, i'll look.
Thanks!

Reply:
Ah right, it already existed :)